### ISSUE TITLE
DetailsList Screenreader Narrator

### ISSUE THREAD

**Issue by kiranmayee18** [OfficeDev/office-ui-fabric-react#10168]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out if relevant)

### Describe the issue:
The sort order of a column header is not announced by the narrator screenreader on edge. (It is announced in NVDA+chrome)

#### Actual behavior:

Narrator does not announce any information about the sorting order of a column header.

#### Expected behavior:

Narrator should  announce " Information about the sorting order of the column header" like sorted by alphabetically  Ascending order or descending order.

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)

Products/sites affected: (if applicable)

**Comment by JasonGore** [comment]
The [ARIA reference example](https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html) also does not announce sorting order. 

This is a known and open issue in Narrator.

Related Issues (some duplicates): #6646, #6011, #8676, #9092
